src/data/mv_polynomial/basic.lean

```diff
@@ -125,7 +125,7 @@ variables (R σ)
 theorem algebra_map_eq : algebra_map R (mv_polynomial σ R) = C := rfl
 variables {R σ}
 
-/-- `X n` is the degree `1` monomial `1*n` -/
+/-- `X n` is the degree `1` monomial `X_n` -/
```

Comment:
just checking, the original doc string was wrong? Unless `1*n` is some notation that I'm unaware of